grape-pipeline.nf

```diff
@@ -46,20 +48,51 @@ params.wigRefPrefix = ''
 params.inferExpThreshold = 0.8
 
 // Process channels
-Channel.empty().into {
-    fetchInput
-    fastaIndexGenomes; fastaIndexAnnotations
-    indexGenomes; indexAnnotations
-    mappingInput; mappingIndex; mappingAnnotations
-    txIndexGenomes; txIndexAnnotations
-    mergeBamInput
-    markdupInput
-    inferExpInput; inferExpAnnotations
-    bamStatsInput; bamStatsAnnotations
-    bigwigInput; bigwigFastaIndex
-    contigInput; contigFastaIndex
-    quantificationInput; quantificationTxIndex
-}
+Channel.empty().set { fetchInput }
```

Comment:
we should check if all these channel initialisations are still needed with DSL2

Reply:
So I suppose the channel initialisations are still needed